drivers/watchdog/Kconfig.stm32

```diff
@@ -30,6 +31,25 @@ config IWDG_STM32_INITIAL_TIMEOUT
 	  Limiting maximum timeout to a safe value of 26214 ms here, which was
 	  calculated for highest LSI frequency among STM32 MCUs of 40 kHz.
 
+config IWDG_STM32_EARLY_WAKEUP
+	bool "Early Wakeup Interrupt support for STM32 IWDG"
+	default y
```

Comment:
As most users didn't need it until now, keep default disabled unless there is a good reason to do otherwise.
Drawback is that we need to enable at least once in wwdg tests using a board overlay to be sure it is compiled in CI.

Reply:
Yeah, but has footprint impact.